src/services/api.ts

```diff
@@ -305,9 +307,16 @@ export class YdbEmbeddedAPI extends AxiosWrapper {
         const uiTimeout = 9 * 60 * 1000;
         const backendTimeout = 10 * 60 * 1000;
 
+        const base64 = !settingsManager.readUserSettingsValue(
```

Comment:
Could you please add comment explanation to `base64` param? Something like `encode string query results with base64`

Reply:
Yeah.